Src/BootCamp.Chapter/MakeStuffWithInput.cs

```diff
@@ -0,0 +1,244 @@
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class MakeStuffWithInput
+    {
+        //TODO Make the Stringmakers more streamlined so they can be called and read more easy. and modular.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5123803)